paimon-core/src/main/java/org/apache/paimon/rest/requests/ConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.rest.requests;
+
+import org.apache.paimon.rest.RESTRequest;
+
+import org.apache.paimon.shade.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.paimon.shade.jackson2.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/** Request to get config. */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConfigRequest implements RESTRequest {
```

Comment:
Should it contains warehouse? Or maybe we cannot let user to config warehouse option.

Reply:
The parameter should disable the warehouse because catalogs and warehouses are one-to-one for defined catalogs, so the client should not be able to define a warehouse.

Reply:
As get config use HTTP get is enough, i deleted this class.